src/scss/_feed.scss

```diff
@@ -0,0 +1,19 @@
+.sidebar-decor {
+    width: 0;             
+    height: 0;
```

Comment:
Reduce the amount of custom styling you have, as most of this could be achieved with bootstrap classes. See Bootstrap documentation

Reply:
I fixed this